src/binder/bind/bind_standalone_call_function.cpp

```diff
@@ -0,0 +1,41 @@
+#include "binder/binder.h"
+#include "binder/bound_standalone_call_function.h"
+#include "catalog/catalog.h"
+#include "common/exception/binder.h"
+#include "function/built_in_function_utils.h"
+#include "main/client_context.h"
+#include "parser/expression/parsed_function_expression.h"
+#include "parser/standalone_call_function.h"
+
+using namespace kuzu::common;
+
+namespace kuzu {
+namespace binder {
+
+std::unique_ptr<BoundStatement> Binder::bindStandaloneCallFunction(
```

Comment:
Some of the logic should be refactored with `bindInQueryCall`, e.g. I think there should be a `bindTableFunction` interface.